guides/session.rst

```diff
@@ -123,22 +199,20 @@ useful methods for you to use in your ``teardown()`` methods:
     $session->reset();
 
     // hard-reset:
+    $session->stop();
+    // or if you want to start again at the same time
     $session->restart();
 
-Both methods do exactly the same job for headless browsers, they clear browser's
-cookies and history. The difference appears with browser controllers:
-
-* ``$session->reset()`` will try to clean all available from browser side
-  cookies. It's very fast and doesn't require the physical reload of the
-  browser between tests, making them much faster. But it has a disadvantage:
-  it clears only the cookies available browser-side. And we also have ``http-only``
-  cookies. In such case, resetting simply won't work. Also, browsing history
-  will state the same after this call. So, it's very fast, but limited in
-  complex cases.
+Stopping the session is the best way to reset the session to its initial
+state. It will close the browser entirely. Using the session again requires
```

Comment:
To use the session again, you need to start the session before any other action.